integrations/db/src/utils/sync-game-state.ts

```diff
@@ -44,8 +65,25 @@ export function syncGameState(
     actorModels,
     logger,
     markToResendFullState,
+    electricUrl,
   }: SyncGameStateOptions,
 ) {
+  // If Electric URL is provided, use real-time sync; otherwise fall back to polling
+  if (electricUrl) {
+    return ResultAsync.fromPromise(
+      createElectricSyncSession(drizzle, {
+        area,
+        state,
+        actorModels,
+        logger,
+        markToResendFullState,
+        electricUrl,
+      }),
+      (e) => e,
+    );
+  }
+
+  // Backwards compatible: single sync operation for polling
   return ResultAsync.fromPromise(save().then(load), (e) => e);
```

Comment:
no, i never asked you to keep it backwards compatible. you should completely refactor sync-game-state.ts to be using electric